scriptmodules/libretrocores/lr-vice.sh

```diff
@@ -10,37 +10,57 @@
 #
 
 rp_module_id="lr-vice"
-rp_module_desc="C64 emulator - port of VICE for libretro"
-rp_module_help="ROM Extensions: .cmd .crt .d64 .d71 .d80 .d81 .g64 .m3u .prg .t64 .tap .x64 .zip .vsf\n\nCopy your Commodore 64 games to $romdir/c64"
+rp_module_desc="C64 / C128 / PET / Plus4 / VIC20 emulator - port of VICE for libretro"
+rp_module_help="ROM Extensions: .cmd .crt .d64 .d71 .d80 .d81 .g64 .m3u .prg .t64 .tap .x64 .zip .vsf\n\nCopy your games to $romdir/c64"
 rp_module_licence="GPL2 https://raw.githubusercontent.com/libretro/vice-libretro/master/vice/COPYING"
 rp_module_repo="git https://github.com/libretro/vice-libretro.git master"
 rp_module_section="opt"
 rp_module_flags=""
 
+function _get_targets_lr-vice() {
+    echo x64 x64dtv x64sc x128 xpet xplus4 xvic
+}
+
 function sources_lr-vice() {
     gitPullOrClone
 }
 
 function build_lr-vice() {
-    make clean
-    make
-    md_ret_require="$md_build/vice_x64_libretro.so"
+    mkdir -p "$md_build/cores"
+    local target
+    for target in $(_get_targets_lr-vice); do
+        make clean
```

Comment:
Maybe it is advisable to put `make clean` outside the loop? Cuts compile time by roughly 2/3.

Reply:
I had that originally and it produced issues with the resulting cores. Maybe upstream libretro makefile has been improved, but feel free to test. Upstream mentions needing to clean before also afair.

Reply:
Actually just see my comments on the PR as I've already mentioned it.

Reply:
Here: https://github.com/RetroPie/RetroPie-Setup/pull/3126#issuecomment-623208782

But if anything has changed I can retest. Cheers.

Reply:
Compiled ok on my system but it seems sth is still lurking in the resulting builds: In their [README](https://github.com/libretro/vice-libretro/blame/master/README.md#L195) they note to run `make clean` before each building of an emulator.
So better keep this scriptmodule keep as-is.